pom.xml

```diff
@@ -16,6 +16,12 @@
     <name>Favorite</name>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Favorite+Plugin</url>
 
+    <licenses>
```

Comment:
Please remove all the license attribution changes, including the LICENSE.md and put in another PR until I can get @larrys to tell us the license.